readthedocs/doc_builder/director.py

```diff
@@ -522,6 +523,8 @@ def install_build_tools(self):
                 record=False,
             )
 
+        build_tools_storage = get_storage(self.data, StorageType.build_tools)
```

Comment:
There is no need to pass the whole object here, I would pass only the data we need here.
```suggestion
        build_tools_storage = get_storage(
          build_id=self.data.build["id"],
          project=self.data.project,
          api_client=self.data.api_client,
          StorageType.build_media
        )
```